### PR TITLE
Happy price initial

### PR DESCRIPTION
**Pull request by mvoland**
Owner can create Happy Minute temporary prices
Happy Prices are shown in Item views
Update Seed

### CODE REVIEW

File: app/views/happy_prices/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  <!-- price !-->
+  <div class="form-group">
+    <%= form.label :price, 'Prix' %>
+    <%= form.number_field :price, step: 0.01, min: 0.01, class: "form-control text-primary"  %>
+  </div>
+
+  <!-- quantity !-->
+  <div class="form-group">
+    <%= form.label :price, 'Quantité' %>
```
Comment (Vivien-Ploix) [inline app/views/happy_prices/_form.html.erb:23]: Shouldn't it be "form.label :quantity" here ?

Comment (mvoland) [inline app/views/happy_prices/_form.html.erb:23]:
```suggestion
    <%= form.label :quantity, 'Quantité' %>
```

Comment (mvoland) [reply app/views/happy_prices/_form.html.erb:23]: Yes

**Review by Vivien-Ploix — APPROVED** [review]

**Review by Vivien-Ploix — APPROVED** [review]

**Review by StanislasBASQUIN — APPROVED** [review]

**Comment by StanislasBASQUIN** [comment]
[Conception du Happy Minute](https://trello.com/c/HUTBqJcH/95-conception-du-happy-minute)

**Pull request merged**